ansible_collections/arista/avd/roles/eos_designs/templates/facts/topology/p2p-uplinks.j2

```diff
@@ -12,6 +12,7 @@ topology:
 {%             do topology_peers.append(uplink_switch) %}
     {{ uplink_interface }}:
       peer: {{ uplink_switch }}
+      peer_is_deployed: {{ uplink_switch_facts.is_deployed }}
```

Comment:
would it make sense to have this information included in the topology csv fail?

Reply:
Yes, added this in as we present it in the MD docs, under the column "Provisioned in CloudVision"